src/components/v2/shared/V2CurrencyAmount.tsx

```diff
@@ -14,7 +14,7 @@ export default function V2CurrencyAmount({
 }) {
   if (!amount) return null
 
-  if (currency?.eq(V2_CURRENCY_ETH)) {
+  if (currency?.eq(0) || currency?.eq(V2_CURRENCY_ETH)) {
```

Comment:
This fixes this being blank as described end of description: 

<img width="533" alt="166153550-ebc7a395-7d4a-41c3-b8af-65d0a0ebd573" src="https://user-images.githubusercontent.com/96150256/166307096-92351e40-9142-4076-a249-41d89d938681.png">

Reply:
Hmm, this is a V2 component but we are accounting for a V1 currency value. Is there a different solution here which keeps this component v2-specific?

Reply:
This shouldn't be necessary anymore, but it fixes existing V2 project's that were launched with this incorrect currency value. I've fixed that problem (of projects deploying with currency === 0), but I thought adding this as a safeguard as something similar happening in the future - makes sure V2CurrencyAmounts will always default to ETH rather than not rendering